packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -14,6 +14,11 @@ export type MenuSectionProps = React.PropsWithChildren<{
 	 * The text to display in header of the menu section.
 	 */
 	header: string;
+
+	/**
+	 * The refresh icon to display in the header of the menu section.
+	 */
+	refreshIcon?: React.ReactElement;
```

Comment:
Since this is for a more general `MenuSection` component, we should probably just call this something like "icon" (and update the documentation to be more general). That will make it more clear that we can use this property for other icons if we want to in the future.

Reply:
An alternative approach would be to update `header` to take a ReactElement instead of a string - then consumers can just pass in whatever they want (including the refresh button in our Containers item case).

Reply:
Changed it to `icon`. I think is we make the header as `ReactElement` we might not guarantee every Menu section have a string header as beginning.

Reply:
dang it...I clicked too fast. Can I include the fix in https://github.com/microsoft/FluidFramework/pull/15322?

Reply:
@chentong7 Can we put up a quick follow-up PR to correct these docs?

Reply:
Sure!